error-handlers.js

```diff
@@ -1,4 +1,22 @@
-exports.handleServerErrors = (err, req, res, next) => {
```

Comment:
I would suggest splitting the two types of errors (custom errors - ie promise.rejects) and psql errors into a seperate functions.

Reply:
sorry I'm not quite sure what you mean here. they are already split into handles invalid paths and custom 404s on the changes?